pymc/distributions/continuous.py

```diff
@@ -1898,19 +1714,6 @@ def moment(rv, size, nu, mu, sigma):
         return mu
 
     def logp(value, nu, mu, sigma):
-        """
```

Comment:
Just in general curious why the docstrings are removed

Reply:
Because they were wrong since V4 began